packages/neo-one-client-full-common/src/models/BaseState.ts

```diff
@@ -1,8 +0,0 @@
-export class BaseState {
-  public static readonly VERSION = 0;
-  public readonly version: number;
-
-  public constructor({ version: versionIn }: { readonly version?: number }) {
-    this.version = versionIn === undefined ? (this.constructor as typeof BaseState).VERSION : versionIn;
-  }
-}
```

Comment:
We use this in node-core. In `Account`, `Asset`, `TransactionData`, `Validator`, and `ValidatorsCount`. You may have already covered this in the `csharp`, if so then ignore. It might also make sense to just move this to the node-core package.